### ISSUE TITLE
Wrong Sentinel Register Command

### ISSUE THREAD

**Issue by Gorgeous-Patrick** [Jaseci-Labs/jaseci#124]
**Describe the bug**
The command the registers the sentinel is wrong.

**To Reproduce**
Run in JSCTL:
```
sentinel register -name fam -code fam.jac -set_active true
```

**Expected behavior**
It should create a new sentinel and return JSON.

**Screenshots**
No screenshot available.

**Environment**
 - OS: [e.g. Linux, WSL] Linux Manjaro
 - Jaseci version: [e.g. 1.0, 1.3, latest main]
 - kubernetes engine [e.g. docker-desktop, minikub, aws eks]

**Additional context**

I think it should be 
```
sentinel register -name fam fam.jac -set_active true
```
Will make a PR later..

**Comment by ypkang** [comment]
Good catch @Gorgeous-Patrick. Please make a PR and I will merge :)

**Comment by ypkang** [comment]
Merged with #130
